series-formelles.tex

```diff
@@ -1590,10 +1590,10 @@ \subsection{Divided powers and substitution}
 \pref{thm:card-subst} suggests adopting the notation $M (X)$ to
 designate a species $M$. The variable $X$ is interpreted as the
 \emph{singleton} species: there is only a single structure of the
-species $X$ (up to isomorphism) and it is \trans{supported by the
-  singletons}{port\'ee par les singletons}.  The result $M (X)$ of the
+species $X$ (up to isomorphism) and it is carried by singletons.
+The result $M (X)$ of the
 substitution of $X$ in $M$ is isomorphic to $M$. For some species we
-adopt a \trans{frankly}{franchement} algebraic notation if it does not
+shall adopt readily algebraic notation if it does not
```

Comment:
What does "readily algebraic notation" mean?  Or should that say "we readily adopt algebraic notation"?

Reply:
Here I thought he meant to use algebraic notation *resolutely* and *consistently* (i.e., as opposed to other possible notations), but now I think it's more about reflecting the similarity between the species and their generating functions. A better adverb may thus be something like "*genuinely*".